src/index.js

```diff
@@ -381,14 +383,14 @@ let readScores = (workbook) => {
             leadAddress.c = cells.firstLead.c
             callAddress.c = cells.firstCall.c
             injAddress.c = cells.firstInj.c
-            npAddress.c = cells.firstNp.c           
-        
-            for(let l = 0; l< maxJams; l++){
+            npAddress.c = cells.firstNp.c
+
+            for(let l = 0; l < maxJams; l++){
```

Comment:
As you requested to be mocked for your programming style, geez man, just choose when to use whitespace or not!